packs/feats/divine-castigation.json

```diff
@@ -69,6 +69,30 @@
                 ],
                 "property": "traits",
                 "value": "unholy"
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    "divine-castigation",
+                    {
+                        "or": [
+                            "self:trait:holy",
+                            "self:trait:unholy"
+                        ]
+                    },
+                    {
+                        "or": [
+                            "item:slug:harm",
+                            "item:slug:heal"
+                        ]
+                    }
+                ],
+                "selectors": [
+                    "spell-damage"
+                ],
```

Comment:
```suggestion
```

Reply:
Sorry, don't understand logic of delete it

After it, json would be invalid

Reply:
<img width="369" alt="image" src="https://github.com/user-attachments/assets/1e1c1d4f-06f5-4991-b8e8-fe471940aba7">

Reply:
Oh, right, this is a damage alteration rather than an item one.